semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/HttpIncubatingAttributes.java

```diff
@@ -229,25 +195,15 @@ public final class HttpIncubatingAttributes {
   /**
    * Deprecated, use {@code http.response.body.size} instead.
    *
-   * @deprecated Deprecated, use `http.response.body.size` instead.
+   * <p>
+   *
+   * @deprecated Replace by {@code http.response.body.size}.
    */
   @Deprecated
   public static final AttributeKey<Long> HTTP_RESPONSE_CONTENT_LENGTH_UNCOMPRESSED =
       longKey("http.response_content_length_uncompressed");
 
   /**
-   * The matched route, that is, the path template in the format used by the respective server
-   * framework.
-   *
-   * <p>Notes:
-   *
-   * <ul>
-   *   <li>MUST NOT be populated when this is not supported by the HTTP server framework as the
-   *       route attribute should have low-cardinality and the URI path can NOT substitute it.
-   *       SHOULD include the <a href="/docs/http/http-spans.md#http-server-definitions">application
-   *       root</a> if there is one.
-   * </ul>
-   *
```

Comment:
Why does this description go away? Is it gone from the YAML model? Or it something with the template?

Reply:
It's how I `if-else` the template. I can add them back if you wanted, but thesee are attributes that have moved from Incubating -> Stable so I didn't keep the description.

Reply:
fixed